### PR TITLE
Fix method getTableSize in User bean which is used for table size and remove it from UserService

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/services/data/UserService.java [hunk 1]
```diff
@@ -726,18 +726,6 @@ public boolean isLoginValid(String login, String filePath) {
         return true;
     }
 
-    /**
-     * Table size.
-     *
-     * @return table size
-     */
-    public Integer getTableSize(User user) {
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/services/data/UserService.java]: Why did you move it to the user?

Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/services/data/UserService.java]: I didn't move it. It was already there as it takes data from database. All pages are using user.getTableSize for data tables, so this one was not used at all.
